packages/ui/src/components/EventsCard/EventsCard.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import { Box, BoxProps } from '../Box';
+import { Button } from '../Button';
+import { Link } from '../Link';
+import { Typography } from '../Typography';
+import { BUTTON_TEXT, MAIN_TITLE } from './EventsCard.mocks';
+
+export type Event = {
+  id: number;
+  title: string;
+  date: Date | { from: Date; to: Date };
+  description: string;
+  url: string;
+};
+
+export type EventCardProps = { events: Event[] } & BoxProps;
+
+const addZeroToDateValue = (dateValue: number): string => {
+  if (dateValue === undefined) {
+    return '';
+  }
+
+  return dateValue >= 10 ? `${dateValue}` : `0${dateValue}`;
+};
+
+const getDayAndMonth = (date: Date): string => {
+  if (date === undefined) {
+    return '';
+  }
+
+  return `${addZeroToDateValue(date.getDate())}.${addZeroToDateValue(date.getMonth() + 1)}`;
+};
+
+const getHourAndMinute = (date: Date): string => {
+  if (date === undefined) {
+    return '';
+  }
+
+  return `${addZeroToDateValue(date.getHours())}:${addZeroToDateValue(date.getMinutes())}`;
+};
+
+export const EventsCard = ({ events, ...props }: EventCardProps) => {
+  return (
+    <Box width="100%" borderRadius={8} padding="32px" boxShadow="base" backgroundColor="white" {...props}>
+      <Typography size="xl" color="gray.900" fontWeight={800} lineHeight="28px" mb="24px">
+        {MAIN_TITLE}
+      </Typography>
+      {(events || []).map(({ id, title, date, description, url }: Event, index: number) => (
```

Comment:
🤔 When you want to pass a default value it would be better to do so when restructuring props

Reply:
What do you mean by default value ? I don't use any default value for this props. Should I use ?

Reply:
When you destructure props, use `events = []`